src/changefile/writeChangeFiles.ts

```diff
@@ -13,7 +13,8 @@ export function writeChangeFiles(
   changes: {
     [pkgname: string]: ChangeFileInfo;
   },
-  cwd: string
+  cwd: string,
+  commitChangeFiles?: boolean
```

Comment:
**suggestion:** you can use default args pattern thus change on line 41 can be removed

-> 
```diff
-commitChangeFiles?: boolean
+commitChanges = true  // TypeScript will properly infer that this argument is optional and type of boolean
```

↓

```
if (commitChanges) {
      commit('Change files', cwd);
}
```

Reply:
Excellent, thanks!